lua/scriptwrapper.lua

```diff
@@ -15,38 +15,56 @@ package.path = EngineDir .. "/lua/?.lua"
 local sandbox = require('sandbox')
 local stp = require('stacktraceplus')
 
+local isSandboxed
```

Comment:
Could also be passed as a function argument? 
Or can scripts this way detect if they are sand boxed? Shouldn't it be set in environment for that?

Think it would be important to show a warning message on the console or in the GUI when someone tries to run any downloaded map this way. Have them confirm that they know what the risk is by agreeing to it once (per map that they open).

This feature is VERY dangerous.